Sources/Bitcoin/WitnessProgram.swift

```diff
@@ -12,6 +12,7 @@ public struct WitnessProgram: Equatable {
     public let program: Data
 
     var encoded: Data {
```

Comment:
Should this be optional? If no valid

Reply:
here nil is same semantic as empty Data()